apps/web/src/pages/About/Member/Member.tsx

```diff
@@ -1,33 +1,43 @@
-import Image from 'next/future/image';
+import Image from 'next/image';
 
 import type { MemberType } from '@/pages/About/About';
 import { useTranslation } from 'next-i18next';
 
+import { Floating } from 'ui';
+
 interface MemberProps {
   member: MemberType;
 }
 
 export const Member = ({ member }: MemberProps) => {
   const { t } = useTranslation('about');
   return (
-    <a target='_blank' href={member.html_url} rel='noreferrer'>
-      <div className='flex h-[300px] w-[250px] flex-col items-center rounded-lg bg-grey-dark transition-shadow hover:shadow-xl'>
-        <Image
-          className='h-[200px] rounded-t-lg object-cover'
-          src={member.avatar_url}
-          alt='github-profile'
-          width={250}
-          height={200}
-        />
-        <div className='flex h-full flex-col items-center justify-center gap-1'>
-          <p className='text-center font-mono text-xs'>@{member.login}</p>
-          <p className='text-center font-mono text-4xl'>
-            {member.contributions}
-          </p>
-          <p className='text-center font-mono text-xs'>{t('contributions')}</p>
+    <Floating
+      content={
+        <div className='flex w-full items-center gap-4 rounded-full bg-black py-2 px-4 hover:scale-105 hover:bg-grey-dark'>
+          <p className='text-sm capitalize'>{t('contributions')}</p>
+          <p className='font-bold text-white'>{member.contributions}</p>
+        </div>
+      }
+    >
+      <a
+        target='_blank'
+        href={member.html_url}
+        rel='noreferrer'
+        className='group relative flex w-[250px] cursor-pointer flex-col items-center justify-center gap-4 rounded-md py-8 transition-all duration-150 ease-linear hover:scale-105 hover:bg-grey-dark'
+      >
+        <div className='relative h-[100px] w-[100px] overflow-hidden rounded-full'>
+          <Image
+            src={member.avatar_url}
+            alt='github-profile'
+            layout='fill'
+            objectFit='cover'
+          />
```

Comment:
remove the wrapping div and give static height and width when using future/image

Reply:
done